packages/component-elife-app/server/policies/isAuthor.js

```diff
@@ -0,0 +1,8 @@
+module.exports = async (userId, operation, request, context) => {
```

Comment:
This shouldn't need to be an async function if you're resolving the promise on return. awaiting the calling of this should be enough.

Reply:
Yeah, you're right - because the function returns a promise anyway, there's no need to double wrap it